index.html

```diff
@@ -1330,8 +1330,10 @@ <h2>DID Subject</h2>
 such as when a <a>DID resolver</a> is performing <a>DID resolution</a>.
 However, the fully resolved <a>DID document</a> always contains a valid
 <code><a>id</a></code> property. The value of <code><a>id</a></code> in the
-resolved <a>DID document</a> is expected to match the <a>DID</a> that was
-resolved.
+resolved <a>DID document</a> MUST match the <a>DID</a> that was
+resolved, or be populated according to the <a>DID method</a> with another
+<a>DID</a> it specifies is the equivalent canonical <a>DID</a> for the <a>DID</a> 
+that was resolved.
```

Comment:
```suggestion
resolved <a>DID document</a> MUST match the <a>DID</a> that was
resolved, or be populated with an equivalent, canonical <a>DID</> 
as specified by the related <a>DID method</a>.
```

Reply:
I understood what you were saying and like the change, but the wording felt kind of clumsy, so this is my attempt to wordsmith.
Just a suggestion.

Reply:
Two things here I want to note:

1. Where it says `THE equivalent canonical DID`, was intentional to make sure that it was a singular relationship, where as a change to `an equivalent canonical DID` seems to suggest that it is selecting from many. I think it would be best to leave this `the`.
2. While I am fine with the general change of the wording to `as specified by the related DID method`, do you think we should use a different word that `related` here? I ask because this is about relationships between things, and maybe we should be more explicit that its not just the `related` method, its the Method of the DID being resolved.

Reply:
good points, suggestion has been updated.